tests/common-tests.h

```diff
@@ -4125,7 +4125,7 @@ void check_for_exception_on_truncation(session& sql)
 }
 
 // And another helper for the test below.
-void check_for_no_truncation(session& sql)
+void check_for_no_truncation(session& sql, bool with_padding)
```

Comment:
Could default to `bool with_padding = false` but OTOH, some may prefer explicit argument, especially for booleans.